src/open_inwoner/js/components/anchor-menu/scroll_to_top.js

```diff
@@ -0,0 +1,15 @@
+// Get the button
+const mybutton = document.getElementById('scroll-anchor')
+
+// When the user scrolls down 80px from the top of the document, show the button
+window.onscroll = function () {
+  scrollFunction()
+}
```

Comment:
In general try to use `addEventListener` instead of assigning to (the global) `.on<xyz>` handler, eg: use `window.addEventListener("scroll", .. ` here.

Also I recall there was something about scroll events and performance. I'm not 100% sure it applies here but if I google about it I see stuff about jankiness and using `{ passive: true }` when adding the event listener (and a whole bunch of hacky solutions with intervals).

Reply:
The intervals are good for old browsers, but we support the modern ones with Passive event listener now :)

Reply:
Very cool.